container_pipeline/lib/default_settings.py

```diff
@@ -11,6 +11,9 @@
 LOG_PATH = '/srv/pipeline-logs/cccp.log'
 SERVICE_LOGFILE = "service_debug.log"
 
+LOGS_URL_BASE = "https://registry.centos.org/pipeline-logs/"
+LOGS_DIR = "/srv/pipeline-logs/"
```

Comment:
`LOGS_DIR` name is used at multiple places for exact location of logs directory, since the value referenced is parent dir of exact location, IMO, we should rename it to `LOGS_DIR_BASE`. I have used the same notion when refactoring my part of work https://github.com/rtnpro/container-pipeline-service/pull/5/files#diff-8604b65e1f7dc045676f76f62ef7ed36R19